SmartPlaces.Facilities/lib/IngestionManager/src/IngestionProcessorBase.cs

```diff
@@ -370,7 +370,8 @@ protected void AddComponent(JsonElement targetElement, Dictionary<string, object
                         if (outputSourceDtmi != null && TargetObjectModel.TryGetValue(outputSourceDtmi, out var model))
                         {
                             var relationship = ((DTInterfaceInfo)model).Contents.FirstOrDefault(p => p.Value.EntityKind == DTEntityKind.Relationship && p.Value.Name == outputRelationship.Item1);
-                            var relationshipId = $"{outputSourceDtmi}-{targetDtId}-{outputRelationship.Item1}";
+                            var relationshipId = outputRelationship.Item2 ? $"{targetDtId}-{sourceElementId}-{outputRelationship.Item1}" :
```

Comment:
Out of scope for this PR, but a #GoodFirstTask may be turning the GetOutputRelationship into a named tuple so the reader can tell what Item1 and Item2 are in this case